x-pack/plugins/monitoring/server/lib/logstash/get_node_info.test.ts

```diff
@@ -8,25 +8,35 @@
 import moment from 'moment';
 import { STANDALONE_CLUSTER_CLUSTER_UUID } from '../../../common/constants';
 import { handleResponse, getNodeInfo } from './get_node_info';
+import { LegacyRequest } from '../../types';
 import { standaloneClusterFilter } from '../standalone_clusters/standalone_cluster_query_filter';
 
 describe('get_logstash_info', () => {
-  // TODO: test was not running before and is not up to date
-  it.skip('return undefined for empty response', () => {
-    const result = handleResponse({});
-    expect(result).toBe(undefined);
-  });
-
   it('return mapped data for result with hits, availability = true', () => {
     const result = handleResponse({
       hits: {
+        total: {
+          value: 1,
+        },
         hits: [
           {
+            _index: 'index',
             _source: {
+              cluster_uuid: '123',
+              timestamp: '2021-08-31T15:00:26.330Z',
               logstash_stats: {
                 timestamp: moment().format(),
                 logstash: {
+                  pipeline: {
+                    batch_size: 2,
+                    workers: 2,
+                  },
                   host: 'myhost',
+                  uuid: 'd63b22f8-7f77-4a23-9aac-9813c760e0e0',
```

Comment:
This has me wondering what we could do to reduce the duplication of untested values. I looked at maybe doing a baseResponse object & spread that into handleResponse but it doesn't satisfy types that way. We can roll with this but figured I should ask incase anyone knew a better way.

Reply:
I added a helper function remove the duplication when creating the response object